elements/lisk-blocks/src/state_store/account_store.ts

```diff
@@ -75,7 +75,7 @@ export class AccountStore {
 		);
 
 		if (element) {
-			return cloneDeep(element);
+			return new Account(element.toJSON());
```

Comment:
it might not be relevant to this PR but the variable name `element` sounds weird, it's an account?

Reply:
we can do it in a separate PR as it also requires to be consistent with other stateStore.